src/gui/properties/peerlistwidget.cpp

```diff
@@ -67,7 +67,7 @@ PeerListWidget::PeerListWidget(PropertiesWidget *parent)
     setSelectionMode(QAbstractItemView::ExtendedSelection);
     header()->setStretchLastSection(false);
     // List Model
-    m_listModel = new QStandardItemModel(0, PeerListDelegate::COL_COUNT);
+    m_listModel = new QStandardItemModel(0, PeerListDelegate::COL_COUNT, this);
```

Comment:
Can the same be done in transferlistwidget.cpp?
Also i noticed you removed the hotkey as well, is this correct? Asking for my PR.